ui/app/src/state/reducers/pathNavigatorTree.ts

```diff
@@ -212,9 +212,16 @@ const reducer = createReducer<LookupTable<PathNavigatorTreeStateProps>>(
         ...state[id].offsetByPath,
         [path]: 0
       };
+      // When the item contains a levelDescriptor, it is rendered as another child. When children length equals or
+      // surpasses the limit, the amount of children rendered is the limit + 1, to account for the levelDescriptor.
+      // So, when collapsing and doing a splice of the childrenByParentPath, we need to consider the levelDescriptor.
+      const containsDescriptor = state[id].childrenByParentPath[path].some((path) =>
+        path.includes('crafter-level-descriptor.level.xml')
```

Comment:
I don't believe the level descriptor is necessarily called this way. @sumerjabri can it be a different name?

Maybe we need to account for it from the moment the fetch payload arrives.

@sumerjabri if the back won't account for the lev desc being on its own, should we just put it back with the children? This logic is not ideal.

Reply:
Per decision, collapse logic to reset state was removed to avoid repeating children calls